src/newSeed/stage2.ts

```diff
@@ -7,6 +7,15 @@ export class Stage2 extends BaseStage {
     return re.test(String(file).toLowerCase());
   }
 
+  private isLinkNotBroken(url: string): boolean {
```

Comment:
I am getting this for a valid url.

![image](https://user-images.githubusercontent.com/1821666/123422271-3281e780-d57b-11eb-8269-6c5f99a38433.png)

Reply:
I'm getting the same error for an invalid URL, but I should not be getting an error at all until the user moves on from the page.

![image](https://user-images.githubusercontent.com/1821666/123422440-7117a200-d57b-11eb-8adb-b2679f6f3292.png)